examples/sim_fibo_ad_cp.cpp

```diff
@@ -206,6 +207,38 @@ try
                                   *props, deck, gravity[2], state);
     }
 
+    std::shared_ptr<BlackoilState> distributed_state=Dune::stackobject_to_shared_ptr(state);
+    std::shared_ptr<Opm::BlackoilPropsAdFromDeck> distributed_props=new_props;
+    Dune::CpGrid distributed_grid = *grid;
+    // At this point all properties and state variables are correctly initialized
+    // If there are more than one processors involved, we now repartition the grid
+    // and initilialize new properties and states for it.
+    if( grid->comm().size()>=1 )
+    {
+        if(!param.getDefault("output_vtk", true))
+        {
+            OPM_THROW(std::logic_error, "We only support vtk output during parallel runs");
```

Comment:
This seems to *require* vtk output in a parallel run, rather than guard against other output choices, shouldn't we instead check that we do not ask for other output formats?